.prettierrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
What is this one for?

Reply:
this is for enabling auto-formatting in this repository